src/solc_0.8/defi/rules/ContributionRules.sol

```diff
@@ -11,6 +11,9 @@ import {IERC1155} from "@openzeppelin/contracts-0.8/token/ERC1155/IERC1155.sol";
 contract ContributionRules is Ownable {
     using Address for address;
 
+    uint256 public erc721MultiplierLimit;
+    uint256 public erc1155MultiplierLimit;
```

Comment:
If i'm not mistaken, the limits will be at 0 by default (meaning it will disable the multipliers)
Should we initialize it to ~uint ?

Reply:
done -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/627/commits/b391428f65f735bf5b88789e0b9702049934fb4b